src/Compilers/Core/Portable/PEWriter/Constants.cs

```diff
@@ -29,45 +30,22 @@ internal static class Constants
 
         // Non-portable CompilationRelaxations value:
         public const int CompilationRelaxations_NoStringInterning = 0x0008;
+
+        public const TypeAttributes TypeAttributes_TypeForwarder = (TypeAttributes)0x00200000;
     }
 
-    internal enum TypeFlags : uint
+    internal struct ExportedType
```

Comment:
It doesn't feel like this type belongs in Constants.cs. Also, please add a doc comment for the type.